epics-team-builder/src/reducers/setupPageReducer.js

```diff
@@ -25,6 +26,8 @@ const projects = (state = [], { type, payload }) => {
 
 const projectByStudentId = (state = {}, { type, payload }) => {
   switch (type) {
+    case ASSIGN_PROJ_TO_STUD:
+      return payload;
```

Comment:
You will most likely have to add to the mapping here. If payload is setup correct you can simply do
return {
   ...state,
   ...payload
}